### PR TITLE
Release matlab-whole-body-simulator 3.0.0

### PR DESCRIPTION
**Pull request by traversaro**
fyi @VenusPasandi , if you think there is some relevant change to document in the changelog feel free to mention it.

Fix https://github.com/ami-iit/matlab-whole-body-simulator/issues/73 .

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0] - 2022-04-14
+
+### Added
+- Add support for simulating robots with cylindrical or spherical robots (https://github.com/ami-iit/matlab-whole-body-simulator/pull/71)
```
Comment (fabiodinatale) [inline CHANGELOG.md:12]:
```suggestion
- Added the possibility of simulating robots with cylindrical or spherical feet (https://github.com/ami-iit/matlab-whole-body-simulator/pull/71)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Add a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
```
Comment (fabiodinatale) [inline CHANGELOG.md:13]:
```suggestion
- Add a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and of defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
```

**Review by fabiodinatale — CHANGES_REQUESTED** [review]

**Comment by traversaro** [comment]
@fabiodinatale thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,8 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.0.0] - 2022-04-14
 
 ### Added
-- Add support for simulating robots with cylindrical or spherical robots (https://github.com/ami-iit/matlab-whole-body-simulator/pull/71)
-- Add a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
+- Added the possibility of simulating robots with cylindrical or spherical feet (https://github.com/ami-iit/matlab-whole-body-simulator/pull/71)
+- Add a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and of defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
```
Comment (fabiodinatale) [inline CHANGELOG.md:13]:
```suggestion
- Added a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and of defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
```

**Review by fabiodinatale — APPROVED** [review]
I forgot to suggest also this modification. Sorry!

**Comment by VenusPasandi** [comment]
@traversaro please consider also #75 in the release version.

**Comment by traversaro** [comment]
> @traversaro please consider also #75 in the release version.

Ok thanks.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased]
 
+## [3.0.0] - 2022-04-14
+
+### Added
+- Added the possibility of simulating robots with cylindrical or spherical feet (https://github.com/ami-iit/matlab-whole-body-simulator/pull/71)
+- Added a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and of defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
+
 ### Changed
 - Added the option to load model with the absolute path, without using the YARP resource management at all (https://github.com/ami-iit/matlab-whole-body-simulator/pull/61).
```
Comment (VenusPasandi) [inline CHANGELOG.md:17]: - Added the possibility of interacting with the robot visualizer while the simulation is paused (https://github.com/ami-iit/matlab-whole-body-simulator/pull/63)

**Review by VenusPasandi — APPROVED** [review]

**Review by fabiodinatale — APPROVED** [review]

**Review by VenusPasandi — APPROVED** [review]

**Pull request merged**